src/data/python-quiz.ts

```diff
@@ -1590,6 +1590,15 @@ const pythonQuiz = [
     Explanation: "In Python, list slicing syntax is list[start:stop[:step]]. The start, stop and step parameters are all optional and if omitted will refer to the defaults of 0, length of sequence and 1 respectively.",
     Link: "https://python-reference.readthedocs.io/en/latest/docs/brackets/slicing.html",
   },
+  {
+    Question: "Which Method is use to Remove leading and trailing  whitespace form the String? Eg. convert '    Hello World   ' into 'Hello World' ",
```

Comment:
it is best to use short questions because it will look better rendered on the screen. 

```suggestion
    Question: "In Python, which method is used to trim trailing whitespace from a string?",
```